pkgs/development/libraries/libgudev/default.nix

```diff
@@ -37,6 +39,8 @@ stdenv.mkDerivation rec {
   mesonFlags = [
     # There's a dependency cycle with umockdev and the tests fail to LD_PRELOAD anyway
     "-Dtests=disabled"
+    "-Dintrospection=${if withIntrospection then "enabled" else "disabled"}"
+    "-Dvapi=${if withVala then "enabled" else "disabled"}"
```

Comment:
Maybe I would not even bother with the rest and just do

```suggestion
  ] ++ lib.optionals (stdenv.buildPlatform == stdenv.hostPlatform) [
    "-Dintrospection=disabled"
    "-Dvapi=disabled"
```

Reply:
I changed it to that.